### ISSUE TITLE
Typo in Snips.ai docs 

### ISSUE THREAD

**Issue by letsoulfly** [home-assistant/home-assistant.github.io#6074]
```
utomation:
  garage_door_has_been_open:
    trigger:
     - platform: state
        entity_id: binary_sensor.my_garage_door_sensor
        from: 'off'
        to: 'on'
        for:
          minutes: 10
    sequence:
      service: snips.say_action
        data:
          text: 'Garage door has been open 10 minutes, would you like me to close it?'
          intentFilter:
            - closeGarageDoor
```
https://www.home-assistant.io/components/snips/
It should be **intent_filter** not intentFilter in the say_action part. BTW The description is correct, only the example has this typo.

**Comment by point-4ward** [comment]
Fixed with #6079 when merged 👍

**Comment by fabaff** [comment]
Fixed with https://github.com/home-assistant/home-assistant.io/pull/6079/commits/d235758682ff71f8e265422f6c93f49a816984a6.
